Makefile

```diff
@@ -30,7 +30,7 @@ go_sources=$(call rwildcard,cmd/,*.go) $(call rwildcard,pkg/,*.go) $(call rwildc
 
 # Configure Go
 export GOOS=linux
-export GOARCH=amd64
+export GOARCH=$(shell uname -m | sed 's/x86_64/amd64/;s/aarch64/arm64/')
```

Comment:
due to https://github.com/kubevirt/macvtap-cni/pull/125/files#r1874637326
we might need to support `all` flag as done with other PRs please ?